formstack-consents/src/main/scala/com/gu/identity/formstackconsents/IdentityClient.scala

```diff
@@ -0,0 +1,66 @@
+package com.gu.identity.formstackconsents
+
+import io.circe.syntax._
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent
+import com.gu.identity.formstackconsents.Lambda.{FormstackSubmission, Config}
+import com.typesafe.scalalogging.StrictLogging
+import io.circe.generic.extras._
+import scalaj.http.{Http, HttpResponse}
+import scala.util.Try
+
+class IdentityClient(config: Config) extends StrictLogging {
```

Comment:
It's not really an issue for this lambda, but just a note that this class is doing quite a few things:

- interacting with the identity API
- understanding a Formstack consent and how to transform one to an identity request
- transforming a response from the identity API to an `APIGatewayProxyResponseEvent`
- understanding (statically) what news letters are supported

As an example of why this might be an issue in the context of a larger application: you wouldn't be able to use this class to post a consent from a different source.

My preference would be to break it down into a pure identity API client and a Formstack service, however, it's pretty subjective, so can leave as is if you want.